include/cantera/oneD/Boundary1D.h

```diff
@@ -395,6 +392,8 @@ class ReactingSurf1D : public Boundary1D
     vector_fp m_fixed_cov;
 };
 
+//! @}
+
```

Comment:
When the end of the group is hundreds of lines away, the note as to what group is being ended is helpful (as you have in some other cases).